database/seeder/main.go

```
@@ -0,0 +1,191 @@
+            Slug: fmt.Sprintf("%s-slug", name),
+        }
+        db.Create(&c)
+        cats = append(cats, c)
+    }
+    return cats
+}
+
+func linkCategoriesToPosts(db *gorm.DB, posts []database.Post, cats []database.Category) {
+    for _, post := range posts {
+        n := rand.Intn(len(cats)) + 1
+        db.Model(&post).Association("Categories").Replace(randomSubset(cats, n))
```

golangci-lint (1.64.8)

Error return value of `(*gorm.io/gorm.Association).Replace` is not checked

(errcheck)